styles.css

```diff
@@ -0,0 +1,89 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (max-width: 600px) {
+  nav {
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
+    align-items: center;
+    height: 40px;
+  }
+
+  section {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 10px;
+    padding: 20px;
+    background-image: url("images/background.png");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  h1 {
+    font-weight: 700;
+    line-height: 52px;
+    font-size: 40px;
+    padding-right: 30px;
+  }
```

Comment:
At the moment, your `h1` design is not matching Figma's design regarding the colour used. 
Your implementation | Figma Design
:------:|:------:
![image](https://user-images.githubusercontent.com/83596135/191840414-25df5b2c-aa5f-481d-b62e-4609ae54b2e6.png) | ![image](https://user-images.githubusercontent.com/83596135/191840481-d6a2576b-8871-40a6-8a8f-2e852e6bedb1.png)

- [ ] Please update the colour of this `h1` tag to match the one from the Figma design (the colour used in the Figma design is 
![image](https://user-images.githubusercontent.com/83596135/191841021-bae3ecc6-75de-4d6a-8e34-9d5f14d18992.png))